jgiven-html5-report/src/main/java/com/tngtech/jgiven/report/html5/Html5AttachmentGenerator.java

```diff
@@ -105,6 +112,9 @@ private File writeFile( AttachmentModel attachment, MediaType mediaType ) {
         File targetFile = getTargetFile( attachment.getFileName(), extension );
         try {
             if( attachment.isBinary() ) {
+                File thumbFile = getTargetFile( "attachment-thumb", extension );
+                String thumbnail = compressToThumbnail( attachment.getValue(), extension );
+                Files.write( parseBase64Binary( thumbnail ), thumbFile );
```

Comment:
That does not make sense to me, you first encode the thumbnail to Base64, to immediately decode it afterwards again?